boogiestats/templates/boogie_ui/scorecard.html

```diff
@@ -0,0 +1,172 @@
+    start += d || 0
+    context.fillStyle = '#e08c36'
+    context.fillRect(start, 210, w, 30)
+    start += w || 0
+    context.fillStyle = '#ff0000'
+    context.fillRect(start, 210, m, 30)
+
+    context.fillStyle = '#000000'
+    context.font = '80px Wendy'
+    context.fillText('{{ score.song.chart_info.diff_number }}' || '?', 86, 180)
+    context.font = '30px Wendy'
+    context.fillStyle = '#ffffff'
+    if('{{ score.song.chart_info.steps_type}}'.includes('single')) context.fillText('SINGLE', 81, 123)
+    if('{{ score.song.chart_info.steps_type}}'.includes('double')) context.fillText('DOUBLE', 81, 123)
+    
+    context.font = '50px miso'
+    context.fillText('BoogieStats', 200, 45)
+    context.font = '18px miso'
+    context.fontWeight = '500'
+    context.fillText(`Played by {{ score.player.name }}`, 200, 70)
+    context.fillText('on {{ score.submission_date }}', 200, 85)
+    context.fillText('{{ song.chart_info.pack_name }}', 270, 200)
+    context.font = '30px miso'
+    context.fontWeight = '700'
+
+    const lines = getLines(context, '{{ score.song.chart_info.title }}' || '{{ score.song.hash }}', 260)
+    lines.forEach((line, i) => {
+        context.fillText(line, 270, 170 - (lines.length * 20) + (i * 20))
+    })
+    context.font = '19px miso'
+    context.fillText('{{ score.song.chart_info.artist }}', 270, 170)
+    context.fillText('{{ score.song.chart_info.pack_name }}'.replace('&#x27;', `'`), 270, 190)
+
+    context.font = '30px miso'
+    context.textAlign = 'right'
+
+    if (nodec) {
+        context.fillStyle = 'gray'
+    }
+    context.fillText('DECENT', 190, 400)
+    context.fillText('WAY OFF', 190, 440)
+
+    context.fillStyle = '#ffffff'
+    context.fillText('FANTASTIC', 190, 280)
+    context.fillText('EXCELENT', 190, 320)
+    context.fillText('GREAT', 190, 360)
+    context.fillText('MISS', 190, 480)
+
+    context.font = '50px Wendy'
+    context.textAlign = 'left'
+    context.fillStyle = '#21cce8'
+    context.fillText(`${parseInt('{{ score.fantastics_plus }}') + parseInt('{{ score.fantastics }}')}`, 210, 283)
+    context.fillStyle = '#e29c18'
+    context.fillText('{{ score.excellents }}', 210, 323)
+    context.fillStyle = '#66c955'
+    context.fillText('{{ score.greats }}', 210, 363)
+    context.fillStyle = nodec ? 'gray' : '#6365c9'
+    context.fillText('{{ score.decents }}', 210, 403)
+    context.fillStyle = nodec ? 'gray' : '#e08c36'
+    context.fillText('{{ score.way_offs }} ', 210, 443)
+    context.fillStyle = '#ff0000'
+    context.fillText('{{ score.misses }} ', 210, 483)
+
+    context.fillStyle = '#ffffff'
+    context.font = '140px Wendy'
+    context.textAlign = 'center'
+    context.fillText('{{ score.itg_score|div:100|stringformat:".2f" }}', 200, 570)
+
+    context.font = '40px Wendy'
+    context.textAlign = 'right'
+    context.fillText('EX', 355, 540)
+    context.textAlign = 'center'
+    context.fillText('{{ score.ex_score|div:100|stringformat:".2f" }}', 350, 570)
```

Comment:
the alignment for the EX Score seems to be a bit off:

![image](https://github.com/florczakraf/boogie-stats/assets/9034451/1d707de5-adb9-42dd-b701-9cd5434e32a4)